test/LiquidityManager/invest.test.ts

```diff
@@ -123,12 +157,82 @@ describe.only('LiquidityManager#invest', () => {
         )
     }
 
+    function getRangeTicks(
+        pool: Pool,
+        lowerPricePercentage: number,
+        upperPricePercentage: number,
```

Comment:
we shouldnt use number, you can use bigint and BigNumber

Reply:
This function is only being used for the test, so we don't have to manually input the price, in this way we use the relative current price.